docs/events_and_facts.rst

```diff
@@ -40,7 +40,11 @@ Instead, the ``all`` operator must be used:
 
 
 .. note::
-    To use facts you may use either ``event`` or ``facts`` keys interchangeably.
+    To use facts you may use either ``events`` or ``facts`` keys interchangeably.
```

Comment:
@bzwei Now that we have removed fact and facts, you can only use events. Shouldn't this line get deleted?

Reply:
This work only changes what are passed to playbook as extra_vars. It does not change how fact and facts are used in a rulebook.